docs/services-guide.md

```diff
@@ -4,772 +4,133 @@ toc: true
+
+To dynamically provision a new Kubernetes target cluster 
+see the Stacks Guides for [GCP][stack-guide-gcp], 
+[AWS][stack-guide-aws], and [Azure][stack-guide-azure].
+
+## Table of Contents
+1. [Introduction](#introduction)
+1. [Secure network connectivity for cloud services](#secure-network-connectivity-for-cloud-services)
+1. [Dynamic provisioning with claims and classes](#dynamic-provisioning-with-claims-and-classes)
+1. [Connection secret for pods in a deployment](#connection-secret-for-pods-in-a-deployment)
+1. [Next Steps](#next-steps)
+1. [Learn More](#learn-more)
+
+## Introduction
+Cloud service provisioning can be added to existing clusters by installing 
+Crossplane directly onto the target cluster. Crossplane is designed to 
+integrate with existing cloud networking and security resources, so 
+managed services like RDS, CloudSQL, and Azure DB can be provisioned 
+using Kubernetes objects securly consumed by pods in a cluster. 
+
+Crossplane achieves this by:
+1. establishing secure network connectivity between the worker nodes in a cluster
+and the cloud services using a set of networking and security resources
+1. populating Kuberentes `Secrets` that pods in a `Deployment` can use to 
+securely access the managed service
+
+## Secure network connectivity for cloud services
+Crossplane currently supports private IP secure connectivity for
+AWS, GCP, and Azure Stacks. Managed services instances
+are made available on the cluster's prviate network(s) so pods can access them.
+Crossplane also supports configuring ingress/egress rules to further restrict 
+allowed network traffic.
+
+While each cloud provider uses different resources for establishing 
+secure connectivity between a Kubernetes cluster (EKS, GKE, AKS) 
+and managed services (RDS, CloudSQL, and Azure DB), the basic pattern is the
+same:
+1. Configure cluster networking
+   * network(s) and subnet(s) - L3 networking for the worker nodes
+1. Enable managed service access: 
+   * private service connection / endpoint - make services available via peering or other
+   * private IP range(s) or subnet group - the private IPs a managed service will get
+   * security groups or network rules - to restrict network traffic
+1. Provision a managed service instance
+   * creates an instance (e.g. RDS, CloudSQL, Azure DB MySQL instance)
+   * assigns a private IP from the private IP range above
+1. Securely use the managed service with secrets
+   * pods on a cluster node can access the managed service via private IP
+
+Crossplane provides Kubernetes objects for all of the above, so you can
+define a secure connectivity model for the managed services you want to make 
+available for self-service provisioning in the cluster using claims and classes.
+
+## Dynamic provisioning with claims and classes
+Crossplane employs a layered architecture consisting of managed resources that
+represent a cloud service, and resource claims and classes that enable dynamic
+provisioning on top.
```

Comment:
Maybe something like "enable portable dynamic provisioning of those services".

Reply:
When writing the getting started guide I felt like we use a lot of terminology that may not make sense to newcomers, and wondered whether we should avoid that terminology in overviews, headings, etc. Terms like "claim", "dynamic provisioning" and even perhaps "class" don't seem intuitive to me. I tried to address this by either avoiding those terms in introductory contexts or trying to use them in contexts that might suggest what they mean.